client/components/inline-help-happychat/style.scss

```diff
@@ -0,0 +1,34 @@
+.button.inline-help-happychat {
+	z-index: z-index( 'root', '.floating-help' );
+	margin: 8px;
+	line-height: 0;
+	padding: 2px;
+	border-radius: 100%;
+	background: $blue-wordpress;
+	box-shadow: 0 2px 6px rgba( 0, 0, 0, 0.15 );
+	border: 1px solid darken( $blue-wordpress, 5 );
+
+	.gridicon {
+		fill: $white;
+		margin: 2px;
+		top: 0 !important;
```

Comment:
I suppose there wasn't a straightforward alternative to the `!important`, like a more specific selector?

Reply:
Ahh yes, I did this a while ago lazily! Good spot, I'll fix it up :)